From 0ed6552ab4684acae272bc3d0969876ca169b8ec Mon Sep 17 00:00:00 2001
From: Luiz Capitulino <lcapitulino@redhat.com>
Date: Thu, 10 Jul 2014 14:15:22 -0400
Subject: [redhat] Kconfig: enable CONFIG_CMA

Message-id: <1405001727-22802-4-git-send-email-lcapitulino@redhat.com>
Patchwork-id: 85658
O-Subject: [RHEL7.1 PATCH 3/8] [redhat] Kconfig: enable CONFIG_CMA
Bugzilla: 996763
RH-Acked-by: Rik van Riel <riel@redhat.com>
RH-Acked-by: Andrea Arcangeli <aarcange@redhat.com>

Signed-off-by: Luiz Capitulino <lcapitulino@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/x86_64/CONFIG_CMA b/redhat/configs/generic/x86_64/CONFIG_CMA
new file mode 100644
index 0000000..309c9e7
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_CMA
@@ -0,0 +1 @@
+CONFIG_CMA=y
-- 
1.7.1